htdocs/fourn/class/fournisseur.product.class.php

```diff
@@ -1091,12 +1091,15 @@ public function displayPriceProductFournisseurLog($productFournLogList = array()
 	 *
 	 *	@param	int		$withpicto					Include picto in link (0=No picto, 1=Include picto into link, 2=Only picto)
 	 *	@param	string	$option						On what the link point to ('nolink', ...)
-	 *  @param	int  	$notooltip					1=Disable tooltip
-	 *  @param  string  $morecss            		Add more css on link
+	 *  @param	int		$maxlength					Maxlength of ref
 	 *  @param  int     $save_lastsearch_value    	-1=Auto, 0=No save of lastsearch_values when clicking, 1=Save lastsearch_values whenclicking
+	 *  @param	int		$notooltip					No tooltip
+	 *  @param	int		$add_label					0=Default, 1=Add label into string, >1=Add first chars into string
+	 *  @param	string	$sep						' - '=Separator between ref and label if option 'add_label' is set
+	 *  @param  string  $morecss            		''=Add more css on link
 	 *	@return	string								String with URL
 	 */
-	public function getNomUrl($withpicto = 0, $option = '', $notooltip = 0, $morecss = '', $save_lastsearch_value = -1)
+	public function getNomUrl($withpicto = 0, $option = '', $maxlength = 0, $save_lastsearch_value = -1, $notooltip = 0, $add_label = 0, $sep = ' - ', $morecss = '')
```

Comment:
Changing position of parameters will break the compatibility.

Reply:
@eldy 
I know that it's dangerous but getNomUrl method seems not to be used with the 3rd parameter.
I prefer to rewrite this method now with sames paramaters as Product class.
Do you think we can keep this method incompatible with Product class ?